### DISCUSSION TITLE
About simulating a missile

### DISCUSSION THREAD

**Discussion by mrwangyou** [JSBSim-Team/jsbsim#677] [Q&A]
Hello everybody,

Does anyone know whether I can launch a missile model with JSBSim? I want to simulate the situation when a plane is locked-on by a missile. Can I use JSBSim to simulate a missile just like simulating a plane? Does JSBSim have a missile model?

Thanks!

**Comment by seanmcleod** [comment]
JSBSim is a full 6-DOF simulator so you can definitely model a missile. The JSBSim repo itself doesn't appear to have any missile models. But I'm sure of you look online you'll find some.

**Reply by mrwangyou** [reply]
Thank you for your advice. I'll check the online repo :) So do you know any online websites where I can find somethings related to missiles?
